pom.xml

```diff
@@ -853,6 +853,7 @@
           --add-opens java.base/java.util=ALL-UNNAMED <!--System Lambda-->
           --add-opens java.base/java.util.concurrent=ALL-UNNAMED <!--PowerMock-->
           --add-opens java.base/sun.net=ALL-UNNAMED <!--netty.DnsResolverUtil-->
+          --add-opens java.base/java.time=ALL-UNNAMED <!-- java.time.Duration -->
```

Comment:
Why this option is needed?

Reply:
using java.time.Duration in the test so had to add this.

Reply:
Why? `java.time.Duration` is a public exported class. Can you provide a error statck without this option?

Reply:
```
java.lang.reflect.InaccessibleObjectException: Unable to make private void java.time.Duration.readObject(java.io.ObjectInputStream) throws java.io.InvalidObjectException accessible: module java.base does not "opens java.time" to unnamed module @50f8360d
        at java.base/java.lang.reflect.AccessibleObject.throwInaccessibleObjectException(AccessibleObject.java:387)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:363)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:311)
        at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:201)
        at java.base/java.lang.reflect.Method.setAccessible(Method.java:195)
        at org.powermock.reflect.internal.WhiteboxImpl.doGetAllMethods(WhiteboxImpl.java:1508)
        at org.powermock.reflect.internal.WhiteboxImpl.getAllMethods(WhiteboxImpl.java:1482)
        at org.powermock.reflect.internal.WhiteboxImpl.getMethods(WhiteboxImpl.java:1750)
        at org.powermock.reflect.internal.WhiteboxImpl.getMethods(WhiteboxImpl.java:1789)
        at org.powermock.reflect.internal.WhiteboxImpl.getBestMethodCandidate(WhiteboxImpl.java:1008)
        at org.powermock.core.MockInvocation.findMethodToInvoke(MockInvocation.java:58)
        at org.powermock.core.MockInvocation.init(MockInvocation.java:35)
        at org.powermock.core.MockInvocation.<init>(MockInvocation.java:22)
        at org.powermock.core.MockGateway.doMethodCall(MockGateway.java:155)
        at org.powermock.core.MockGateway.methodCall(MockGateway.java:138)
        at org.apache.kafka.clients.consumer.PulsarKafkaConsumerTest.testCommitSync(PulsarKafkaConsumerTest.java:113)
        at org.apache.kafka.clients.consumer.PulsarKafkaConsumerTest_$$_jvst164_0._d8testCommitSync(PulsarKafkaConsumerTest_$$_jvst164_0.java)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:578)
        at org.powermock.modules.testng.internal.PowerMockTestNGMethodHandler.invoke(PowerMockTestNGMethodHandler.java:50)
        at org.apache.kafka.clients.consumer.PulsarKafkaConsumerTest_$$_jvst164_0.testCommitSync(PulsarKafkaConsumerTest_$$_jvst164_0.java)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:578)
        at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
        at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:45)
        at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:73)
        at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1623)

```

Reply:
Thank you! Then it's about reflections and powermock..We may try to reduce such usage later.